src/client/theme-default/components/NextAndPrevLinks.vue

```diff
@@ -35,7 +36,8 @@ export default defineComponent({
     return {
       hasLinks,
       prev,
-      next
+      next,
+      withBase,
```

Comment:
Another option would be to use `withBase` in `getFlatSidebarLinksFromArray`:

```js
    if (item.link) {
      links.push({ text: item.text, link: withBase(item.link) })
    }
```

However, that would also require using `withBase` in `currentPath` for the comparison in `currentIndex` to work as expected.

Calling it in the component seemed consistent with how the logo is currently being rendered.

Reply:
Yeah, I would like to refactor the use of `withBase`, but for now it's being used everywhere so I think this is fine!